routes/web.php

```diff
@@ -89,8 +89,8 @@
 // Route::patch('car/{id}', [CarController::class, 'restore'])->name('car.restore');
 
 
-//classes
-
+//changed task 7
+// i think all resource controller methoda only can have model instance as a paremeter
```

Comment:
hello Safaa,,

Check out [this link](https://medium.com/@moumenalisawe/mastering-route-model-binding-in-laravel-a-comprehensive-guide-d95d3e0327f2) not all resource controller only have access to model instance as parameter

try to define a new method and it will be able to take model instance also

the problem here is coming from softDeletes, softDeletes make the model retrieve only where deleted_at column is null so the solution can be done by explicit model binding where you can define a custom sql to retrieve the data